CHANGELOG.md

```diff
@@ -5,7 +5,23 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 This project uses a version scheme based on the Cadenza main version in the format x.x.y, where x.x is the Cadenza main version and y a functional change or bugfix.
 
 ## Unreleased
-
+### Changed
+- `CadenzaEditGeometryUpdateEvent`. 
+  - Updated the type definition for the `editGeometry:update` event.
+    - The event's `detail` payload has been changed from `{ geometry: Geometry }` to `FeatureCollection | Feature | undefined`, to bring them in line with the rest of our API.
```

Comment:
Needs resolution / answers:
* Why undefined instead of an empty collection?

Proposals:
* This is a breaking change. I'd suggest to mark them specifically, now that we do not have semantic versioning anymore, e.g. using a **BREAKING CHANGE** prefix on the bullet point.
* "Changed" and "Updated" and ChangeEditGeometryUpdate and editGeometry:update are redundant. Proposal:
> The `CadenzaEditGeometryUpdateEvent#detail` is now typed as `FeatureCollection | Feature | undefined` to align it with the rest of the API. (Before: `Geometry`)
* Please check that this indentation actually ends up in the [Releases](https://github.com/DisyInformationssysteme/cadenza.js/releases/tag/v10.2.2) (which is what customers look at).

Reply:
> Why undefined instead of an empty collection?
Since FeatureCollection is a special usecase and will not occur in case of multi-geometries, it does not make sense to use them if the multi-geometry feature is absent.